src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2672,13 +2724,25 @@ export function getMolecularProfileOptions(
                 )
                 .uniq()
                 .value();
-            return {
-                value: value,
-                count: uniqueProfiledSamples.length,
-                label: profiles[0].name,
-                description: profiles[0].description,
-                sampleUniqueKeys: uniqueProfiledSamples,
-            };
+
+            if (withDataType) {
```

Comment:
kinda repetative here.  is there a reason that we cannot always return dataType even if it doesn't exist and is undefined?

Reply:
we can do that, I can update this.